app/src/main/java/com/example/farmeraid/data/QuotasRepository.kt

```diff
@@ -54,10 +56,31 @@ class QuotasRepository {
             return null
         }
     }
-    fun addQuota(market: MarketModel.Market, produce: List<QuotasRepository.ProduceQuota>) {
-        db.collection("quotas").document(market.id)
-            .update("produce", produce.associate {
-                it.produceName to it.produceGoalAmount
-            })
+    suspend fun addQuota(market: MarketModel.Market, produce: List<QuotasRepository.ProduceQuota>) {
+        if (db.collection("quotas").document(market.id).get().await().exists()) {
```

Comment:
Do we want to add the ResponseModel to this? Or do it in a diff CR

Reply:
Yeah I was originally thinking about this. But since we were thinking about doing in a diff PR, I decided to leave it out. But what do you think?